README.rst

```diff
@@ -62,6 +62,20 @@ Xandikos has been tested and works with the following CalDAV/CardDAV clients:
 - `Calendarsync <https://play.google.com/store/apps/details?id=com.icalparse>`_
 - `Tasks <https://github.com/tasks/tasks/tree/caldav>`_
 
+Client instructions
+===================
+
+Some clients can automatically discover the calendar and addressbook URLs from
+a DAV server. For such clients you can simply provide the URL to Xandikos directly.
+
+Clients that lack such automated discovery require the direct URL to a calendar
+or addressbook. One such client is Thunderbird lightning in which case you
+should provide a URL similar to the following:
```

Comment:
Thanks - I was worried this was a Xandikos bug rather than a missing feature in Thunderbird/Lightning, but this page confirms it: http://sabre.io/dav/clients/thunderbird/